code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -51,7 +51,7 @@
 /obj/item/weapon/cartridge/Initialize()
 	. = ..()
 	var/obj/item/device/pda/pda = loc
-	if(istype(pda, /obj/item/device/pda))
+	if(istype(pda))

```

Comment:
I don't actually like this kind of ```istype()``` being used on local variables (I like it for arguments only), I will have to hit @Cyberboss with a stick.